### ISSUE TITLE
Focus Visible 2.4.7 - Element not highlighted on focus

### ISSUE THREAD

**Issue by atsea** [UDelIT/udthemebrand#43]
There are 58 occurrences in the report. UDIT will address 8 occurrences (see  #42 details).

Add this css for _**brand header links, footer social links and footer legal links**_ to receive focus. CSS is already there so we'll see if this fixes the error.

`a:focus,  :focus, .ud-norm--header > a, #ud-id-site--title > a, .ud-norm--footer > a { outline: 1px dotted;}`

**Comment by atsea** [comment]
Added to _wcag.css for testing.

**Comment by atsea** [comment]
fixed and tested on dev.udtbp2.pantheon. outline css removed and replaced with black slightly transparent background.
